File: Model/routes/comanda.js
const express = require('express');
var router = express.Router();
const comanda = require('../controllers/comanda');

router.get("/comanda", comanda.list);
router.get("/comanda/:id_pedido/:codigo_ticket/:codigo_empleado", comanda.get);
router.post("/comanda", comanda.add);
router.put("/comanda", comanda.update);
router.delete("/comanda/:id_pedido/:codigo_ticket/:codigo_empleado", comanda.delete);

module.exports = router;

File: Controller/Modelo/Comanda.cs
namespace Intermodular_MVC_VladimirIriarte.Modelo
{
    public class Comanda
    {
        public int id_pedido { get; set; }
        public int codigo_ticket { get; set; }
        public string dni_empleado { get; set; }
        public string fecha { get; set; }

        public Comanda(int id_pedido, int codigo_ticket, string dni_empleado, string fecha)
        {
            this.id_pedido = id_pedido;
            this.codigo_ticket = codigo_ticket;
            this.dni_empleado = dni_empleado;
            this.fecha = fecha;
        }
    }
}